charts/misc-apps/README.md

```diff
@@ -48,7 +47,6 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | prometheusMsteams.chart | string | `"prometheus-msteams"` | Chart |
 | prometheusMsteams.destination.namespace | string | `"infra-prometheus-msteams"` | Namespace |
 | prometheusMsteams.enabled | bool | `false` | Enable prometheus-msteams |
-| prometheusMsteams.repoURL | string | `"https://prometheus-msteams.github.io/prometheus-msteams"` | [repo](https://prometheus-msteams.github.io/prometheus-msteams) |
```

Comment:
this broke too

Reply:
the repoURL was missing a description which i added in ff284f8a897b501bbd81fb6753984befdf98a0e8